lms/envs/common.py

```diff
@@ -395,6 +395,9 @@
     # Course discovery feature
     'ENABLE_COURSE_DISCOVERY': False,
 
+    # Setting for overriding default filtering facets in Course discovery
+    # COURSE_DISCOVERY_FILTERS = ["org", "language", "modes"]
```

Comment:
I just placed that there as an example on how to override COURSE_DISCOVERY_FILTERS. By default, `["org", "language", "modes"]` is set on the edx-search side.

Someone can uncomment this and remove one or more filter facets to disable some of them, for example "modes" in the case you mentioned yesterday.